project/ISSUE-TRIAGE.md

```diff
@@ -31,53 +28,36 @@ If the author provides the standard information but you are still unable to tria
 If the author does not respond requested information within the timespan of a week, close the issue with a kind note stating that the author can request for the issue to be
 reopened when the necessary information is provided.
 
-### Step 2: Apply the template
-
-When triaging, use the standard template below. You should cut and place the template in the issue's description. 
-The template helps other reviewers find key information in an issue. For example, using a template saves a 
-potential contributor from wading though 100s of comments to find a proposed solution at the very end.  When adding 
-the template to the issue's description also add any required labels to the issue for the classification and difficulty.
-
-Here is a sample summary for an [issue](https://github.com/docker/docker/issues/10545).
-
-```
-**Summary**: docker rm can return a non-zero exit code if the container does not
-exist and it is not easy to parse the error message.
-
-**Proposed solution**:
+### 2. Classify the Issue
 
-docker rm should have consistent exit codes for different types of errors so
-that the user can easily script and know the reason why the command failed. 
+An issue can have multiple of the following labels.
 
-```
-
-### Step 3: Classify the Issue
-
-Classifications help both to inform readers about an issue's priority and how to resolve it.
-This is also helpful for identifying new, critical issues.  "Kinds of" are
-applied to the issue or pull request using labels.  You can apply one or more labels.
-
-
-Kinds of classifications:
+#### Issue kind
 
 | Kind             | Description                                                                                                                     |
 |------------------|---------------------------------------------------------------------------------------------------------------------------------|
-| kind/enhancement | Enhancement are not bugs or new features but can drastically improve usability or performance of a project component.           |
-| kind/cleanup     | Refactoring code or otherwise clarifying documentation.                                                                         |
-| kind/content     | Content that is not documentation such as help or error messages.                                                               |
-| kind/graphics    | Work involving graphics skill                                                                                                   |
-| kind/regression  | Regressions are usually easy fixes as hopefully the action worked previously and git history can be used to propose a solution. |
```

Comment:
A regression could be the result of a bug, but also a regression in functionality that we haven't thought of when making changes. "Something a user was able to do previously, but no longer is able to do". Sometimes it could turn out to be a bug, sometimes not, so I'm with you that it may be an "intermediate" label ("needs investigating"). idk.